src/sagemaker_containers/_env.py

```diff
@@ -222,7 +222,7 @@ class _Env(_mapping.MappingMixin):
     def __init__(self):
         current_host = os.environ.get(_params.CURRENT_HOST_ENV)
         module_name = os.environ.get(_params.USER_PROGRAM_ENV, None)
-        module_dir = os.environ.get(_params.SUBMIT_DIR_ENV, None)
+        module_dir = os.environ.get(_params.SUBMIT_DIR_ENV, 'file:///opt/ml/code')
```

Comment:
Avoid using slash as directory separator because it does not work in all file systems. Use os.path.join

Reply:
Do we need to have **file** in the front? My reason is that it is requires to have remove the prefix file before opening the content.

Reply:
I was following the convention set forward here, https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/job.py#L104, for local mode support within the python sdk.

Reply:
The code that invokes this URI does drop the prefix. Good point on the directory separator.